browser-test/src/applicant/navigation/application_navigation_flow.test.ts

```diff
@@ -92,13 +92,24 @@ test.describe('Applicant navigation flow', () => {
     })
 
     test.describe(
-      'review page with North Star enabled',
+      'flows with North Star enabled',
       {tag: ['@northstar']},
       () => {
         test.beforeEach(async ({page}) => {
           await enableFeatureFlag(page, 'north_star_applicant_ui')
         })
 
+        test('validate block edit page title', async ({
+          page,
```

Comment:
[non- blocking]- but it's good idea to move this a function in support/* folder and pass in the title as an input parameter. This function will have good reusability in the future.

Reply:
Done